packages/api/test/e2e/promise-tx.spec.js

```diff
@@ -26,7 +26,31 @@ describe.skip('e2e transactions', () => {
 
     return api.tx.balances
       .transfer('12ghjsRJpeJpUQaCQeHcBv9pRQA3tdcMxeL8cVk9JHWJGHjd', 12345)
-      .sign(keyring.dave, nonce)
+      .sign(account, { nonce })
+      .send(({ events, status, type }) => {
+        console.log('Transaction status:', type);
+
+        if (type === 'Finalised') {
+          console.log('Completed at block hash', status.value.toHex());
+          console.log('Events:');
+
+          events.forEach(({ phase, event: { data, method, section } }) => {
+            console.log('\t', phase.toString(), `: ${section}.${method}`, data.toString());
+          });
+
+          if (events.length) {
+            done();
+          }
+        }
+      });
+  });
+
+  t('makes a transfer (sign, then send - compat version)', async (done) => {
```

Comment:
it

Reply:
Cool, testing just started, so actually just picked that one up.